Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM opensuse/tumbleweed
+
+RUN zypper -n install openssh-clients ansible ruby ruby-devel
```

Comment:
Dropping the zypper cache afterward (`zypper -q clean -a`) will make your image smaller.

## Before

```shell
docker images ansible-obs_ansible-obs       
REPOSITORY                TAG       IMAGE ID       CREATED          SIZE
ansible-obs_ansible-obs   latest    2e0b2019d8e7   12 minutes ago   390MB
```

## After
```shell
docker images ansible-obs_ansible-obs
REPOSITORY                TAG       IMAGE ID       CREATED          SIZE
ansible-obs_ansible-obs   latest    c99fd8c22ef6   22 minutes ago   335MB
```

Reply:
ah great tip, thanks!